assets/src/img/ethicalads-logo-blue.svg

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I feel like we shouldn't even be checking these assets into the repo. There should be an env var for a link that contains them. We don't want them licensed under the same license as the code, and we definitely don't want to distribute them, which is the issue we hit with RTD.